docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/checkbox/checkbox.scss

```diff
@@ -5,7 +5,11 @@
 @import '../../00-base/mixins/form';
 
 .civictheme-checkbox {
+  $root: &;
+
   position: absolute;
+  display: flex;
+  align-items: center;
 
   &__element {
```

Comment:
do we need this element?

Reply:
Yes, We are applying CSS classes to input element same as before.